packages/app/src/cli/services/environment.ts

```diff
@@ -63,6 +63,7 @@ interface DevEnvironmentOutput {
   storeFqdn: string
   identifiers: UuidOnlyIdentifiers
   updateURLs: boolean | undefined
+  tunnelPlugin: string | undefined
```

Comment:
This cached state is growing crazy. Can't wait for @amcaplan's presets idea to land and make this state explicit in a preset file in the project.